packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/index.css

```diff
@@ -14,10 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.TraceTimelineViewer {
-  border-bottom: 1px solid #bbb;
-}
-
 .json-markup {
```

Comment:
Other styles seem to be global overrides for external packages. These can be moved to global emotion styles too.